lib/srv/alpnproxy/proxy_test.go

```diff
@@ -505,3 +530,69 @@ func TestMatchMySQLConn(t *testing.T) {
 		})
 	}
 }
+
+func TestProxyPingConnections(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+	log.SetFormatter(utils.NewDefaultTextFormatter(trace.IsTerminal(os.Stderr)))
```

Comment:
Why this test makes some modifications to the global logger?

Reply:
Nice catch. It was left from my debug.